### DISCUSSION TITLE
Unable to create the Apache Iceberg table  using SoAL

### DISCUSSION THREAD

**Discussion by JohnChe88** [aws-samples/spark-on-aws-lambda#47] [General]
Please use this discussion board to discuss this topic.

**Comment by reinhardt1kws** [comment]
Hi John,

Iceberg format gives me these errors: 

Missing org.apache.iceberg.aws.glue.GlueCatalog [java.lang.NoClassDefFoundError: software/amazon/awssdk/services/glue/model/EntityNotFoundException]

pyspark.sql.utils.IllegalArgumentException: Cannot initialize Catalog implementation org.apache.iceberg.aws.glue.GlueCatalog: Cannot find constructor for interface org.apache.iceberg.catalog.Catalog

Did build the docker image with Framework ICEBERG and pyspark 3.2.0. 

Is this generally not working or am I missing something?

Cheers
